### PR TITLE
new alias `Obs` for `Observable

### PR DESCRIPTION
**Pull request by asinghvi17**
cc @SimonDanisch @piever @jkrumbiegel @Datseris - what do you all think of the name?

### CODE REVIEW

File: src/Observables.jl [hunk 1]
```diff
@@ -26,6 +26,11 @@ mutable struct Observable{T} <: AbstractObservable{T}
     Observable{T}(val) where {T} = new{T}([], val)
 end
 
+"""
+Alias of `Observable`.  Lika a `Ref`, but updates can be watched by adding a handler using [`on`](@ref).
```
Comment (Datseris) [inline src/Observables.jl:30]: No reason to duplicate docstrings, makes it harder to maintain in the long run. "Alias of `Observable`." is enough IMO.

**Comment by Datseris** [comment]
I definitely agree, as I already use `Obs` so far in my code when I make my own type of observables, e.g. `ParticleObservable` I immediatelly define `PartObs` as an alias.

**Comment by SimonDanisch** [comment]
I love hate it :D I started using it a lot, but it just sounds so weird and incomplete, and completely masks what it is... So I'd be against it ;) But Observable is also pretty annoying :D Maybe we should just come up with an entirely different name? I dont know... I just accepted my fate and started writing out Observable eveywhere, which isn't so bad with autocomplete

**Comment by travigd** [comment]
There's no reason this can't just be done in "consumer" code.

```
using Observables
const Obs = Observable
...
```

Alternatively, we could create the alias in Observables.jl and just not export it by default so that you have to do
```julia
using Observables
using Observables: Obs
```

**Comment by Datseris** [comment]
Yeah, you both have a point. Probably best to get used to using `Observable` and that user does the stuff on their end, like I do.

**Comment by travigd** [comment]
I'm going to close this. :^)

**Pull request closed without merging**